LICENSE

```diff
@@ -0,0 +1,35 @@
+Copyright (C) 2013-2016 Open Source Robotics Foundation, Inc; CA, USA
+Copyright (C) 2016 Janko Petereit <janko.petereit@gmail.com>
+Copyright (C) 2011-2012 Ethan Rublee <ethan.rublee@gmail.com>
```

Comment:
Yes it would be Willow Garage. We should definitely not be adding names and dates here that not in any file. 

As this file is effectively manually duplicating information, any copyright here should be from some other file in the repo, I believe that the content should not be put here unless we have a fully automated way to aggregate the copyrights from their original locations. It's definitely helpful for users if the information is centralized, but if it gets out of date or inaccurate at all, as duplicate information tends to do, it will actually do a disservice to users who rely on the aggregated information.